flexget/plugins/filter/pending_approval.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget.manager import Session
+from flexget.utils.database import entry_synonym
+from sqlalchemy import Column, String, Unicode, Boolean, Integer
+from flexget import db_schema, plugin
+from flexget.event import event
+
+log = logging.getLogger('pending_approval')
+Base = db_schema.versioned_base('pending_approval', 0)
+
+
+class PendingEntry(Base):
+    __tablename__ = 'pending_entries'
+
+    id = Column(Integer, primary_key=True, autoincrement=True, nullable=False)
+    task_name = Column(Unicode)
+    title = Column(Unicode)
+    url = Column(String)
+    approved = Column(Boolean)
+    _json = Column('json', Unicode)
+    entry = entry_synonym('_json')
+
+    def __init__(self, task_name, entry):
+        self.task_name = task_name
+        self.title = entry['title']
+        self.url = entry['url']
+        self.approved = False
+        self.entry = entry
+
+    def __repr__(self):
+        return '<PendingEntry(task_name={},title={},url={},approved={})>' \
+            .format(self.task_name, self.title, self.url, self.approved)
+
+
+class PendingApproval(object):
+    schema = {'type': 'boolean'}
+
+    @staticmethod
+    def _item_query(entry, task, session):
+        return session.query(PendingEntry) \
+            .filter(PendingEntry.task_name == task.name) \
+            .filter(PendingEntry.title == entry['title']) \
+            .filter(PendingEntry.url == entry['url']) \
+            .first()
+
+    @plugin.priority(-1)
+    def on_task_input(self, task, config):
+        if not config:
+            return
+
+        approved_entries = []
+        with Session() as session:
+            # Let details plugin know that it is ok if this task doesn't produce any entries
+            task.no_entries_ok = True
+
+            for entry in task.entries:
```

Comment:
I'm thinking we should do this at the end of the filter phase, and only add entries that are accepted. At this point, we'd also reject any entries in the task that aren't approved in the db yet.